flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -1412,8 +1416,8 @@ static DynamicType GetBuiltinDerivedType(
   auto iter{
       builtinsScope->find(semantics::SourceName{which, std::strlen(which)})};
   if (iter == builtinsScope->cend()) {
-    common::die(
-        "INTERNAL: The __fortran_builtins module does not define the type '%s'",
+    common::die("INTERNAL: The __fortran_builtins module does not define the "
```

Comment:
Please consider rejoining this message string, even if it's long -- split message strings can become impossible to find in the source.